src/components/OrdersWidget/index.tsx

```diff
@@ -241,6 +246,7 @@ const OrdersWidget: React.FC = () => {
                   sellToken={getTokenById(order.sellTokenId)}
                   buyToken={getTokenById(order.buyTokenId)}
                   order={order}
+                  updateLowBalance={updateLowBalanceFactory(order.id)}
```

Comment:
why factory?

Reply:
Because it returns a function. We talked about this in another function I called a factory.
What would be a better name for a function that returns functions?

Reply:
Ah, I see know